worker/apiserver/manifold_test.go

```diff
@@ -371,6 +372,15 @@ type fakeMultiwatcherFactory struct {
 	multiwatcher.Factory
 }
 
+type stubWatchableDB struct {
+	changestream.EventSource
+	database.TxnRunner
+}
+
+type stubWatchableDBGetter struct {
+	db stubWatchableDB
+}
```

Comment:
So once my PR has landed, we want to DRY this up by using a ConstFactory. If we're repeating this stuff a lot, then it would be good to move it into a testing package.

https://github.com/juju/juju/pull/15974/files#diff-d3bce17a3f70860e569a1086563230122ba928e417f2dfcecaa065df46020f71R73-R84